### DISCUSSION TITLE
Tidak ada missing value name ada satu Kolot yang verisimilitude Nan

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#779] [Q&A] [answered]
Selamat malam, saya mau bertanya dalam mengtreatment missing value atau cek missing value yang mana hasilnya tidal ada missing value, name ada satu Kolom yang isinya NaN, apakah kolom ini bisa dibuang saja apabila dia tidak memiliki hubungan dengan kolom lain?

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Pak @mosesnugraha , terima kasih atas pertanyaannya.
Apabila terdapat kolom yang semua nilainya NaN dan tidak memiliki hubungan dengan kolom yang lain, **sangat disarankan untuk dibuang saja**. Proses ini adalah bagian dari data cleansing. Dengan langkah ini, diharapkan data kita bisa lebih rapi dan bisa didapatkan insight yang benar.

Semoga jawaban ini dapat membantu.

Best regards,
Team algoritma
